assets/section-main-product.css

```diff
@@ -648,7 +694,6 @@ a.product__text {
     right: 5rem;
     top: 2.2rem;
   }
-}
```

Comment:
I don't think this should have been deleted..

Reply:
Hotfixed.